src/snipster/__main__.py

```diff
@@ -1,7 +1,10 @@
-from .db import get_session
+from .db import default_session_factory
 from .models import Language, Snippet
 
-with get_session() as session:
+# Store snippet info outside session context
+snippet_info = None
+
+with default_session_factory.get_session() as session:
```

Comment:
I think we can remove this in favor of importing app from cli and running `app()` = the cli